content/zh/live/3.1-live.md

```diff
@@ -0,0 +1,20 @@
+---
+title: KubeSphere v3.1 开源社区交流会直播回放
+description:
```

Comment:
Is it possible to add the value of `Keywords` and `description` to each .md file? I think they are necessary for the search engine.

Reply:
Yes,it's necessary.I have add them.